readthedocs/docsitalia/models.py

```diff
@@ -284,3 +284,144 @@ def __str__(self):
         return (
             _('{0} for {1}')
             .format(self.get_integration_type_display(), self.publisher.name))
+
+
+@python_2_unicode_compatible
+class AllowedTag(models.Model):
+
+    """
+    Tags allowed to be used for a document.
+
+    If a document contains a tag that does not belong to this table, it
+    will be removed.
+    """
+
+    BASE_TAGS = [
```

Comment:
Non so molto di Python quindi potete anche rispondermi semplicemente "no", ma non si potrebbe mettere la lista in un file `tags-vocabulary.py` (o qualcosa di simile) separato e poi importarlo qui in questo modo?
```
import tags-vocabulary
...
BASE_TAGS  = tags-vocabulary
```

Reply:
@francescozaia Ciao, nessun problema. Ho sistemato nel commit https://github.com/italia/docs.italia.it/pull/282/commits/ad5182b01cdf2f0eaa22cf46ed2eaa71e9102521

Reply:
@pauloxnet Per me è indifferente, ma un modulo esplicito e dedicato mi sembra la soluzione più pulita, anche in vista di eventuali estensioni/aggiunte di liste di tag, da usare altrove nel progetto. Comunque dal mio punto di vista va bene fare in entrambi i modi, si può switchare senza fatica. @francescozaia una tua opinione in merito?

Reply:
Ragazzi, su Python comandate voi! Io ragionavo così perché JSista di formazione... @yakky toglici dall'imbarazzo: modulo separato o `__init__.py`? 😄

Reply:
propendo per il modulo separato perché rende più esplicito l'approccio secondo me